cloudfiles/secrets.py

```diff
@@ -8,21 +8,27 @@
 from .lib import mkdir, colorize
 
 HOME = os.path.expanduser('~')
-CLOUD_VOLUME_DIR = os.path.join(HOME, '.cloudvolume', 'secrets')
-CLOUD_FILES_DIR = os.path.join(HOME, '.cloudfiles', 'secrets')
+
+CV_HOME = os.path.join(HOME, '.cloudvolume')
+CLOUD_VOLUME_DIR = os.environ.get("CLOUD_VOLUME_DIR", CV_HOME)
+CLOUD_VOLUME_SECRETS_DIR = os.path.join(CLOUD_VOLUME_DIR, 'secrets')
+
+CF_HOME = os.environ.get("CLOUD_FILES_DIR", HOME)
+CLOUD_FILES_DIR = os.path.join(CF_HOME, '.cloudfiles')
```

Comment:
Still confusing, Do you really want the environment variable `CLOUD_FILES_DIR` to be `$HOME`, but the python variable `CLOUD_FILES_DIR` to go to `$HOME/.cloudfiles`? Since `CV_HOME=CLOUD_VOLUME_DIR=$HOME/.cloudvolume` Shouldn't the cloud-files counterparts both be `$HOME/.cloudfiles` by default

Reply:
Yep. Fixed.